data.txt

```diff
@@ -0,0 +1 @@
+E,0,project meeting,2000-12-20,2000-12-20
```

Comment:
What is the difference between data.txt and duke.txt?

Reply:
I forgot to delete duke.txt and its folder. Good spot and thanks!